src/libs/CardUtils.ts

```diff
@@ -529,6 +529,17 @@ function hasCardListObject(workspaceAccountID: number, feedName: CompanyCardFeed
     return !!workspaceCards.cardList;
 }
 
+/**
+ * Check if the Expensify Card is fully set up and a new card can be issued
+ *
+ * @param policy the policy object
```

Comment:
Do we need these? They kinda seem redundant here

Reply:
you mean the whole comment or just params?